subgraph/scripts/generateSubgraph.js

```diff
@@ -0,0 +1,16 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import Mustache from "mustache";
+
+import { readFileSync, writeFileSync } from "fs";
+
+const network = process.argv.at(2);
+const template = readFileSync("./subgraph.template.yaml", "utf-8");
+// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
Why did you disable eslint for the file?

Reply:
removed!